neo/sys/win32/win_main.cpp

```diff
@@ -602,7 +602,17 @@ uintptr_t Sys_DLL_Load( const char *dllName ) {
 		}
 	} else {
 		DWORD e = GetLastError();
-		// TODO
+		//when encountering errors, pls ammend to list
+		//see https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes
+		idStr errorStr = va("\n\t\t Unknown Error [%i (0x%X)]",e,e);
+		if (e == 0x7E)
```

Comment:
this error should be ignored - it's ok if the file has not been found (see the POSIX implementation), dhewm3 tries to open them from different folders so it's expected taht sometimes fails

Reply:
also, the code might be more readable when using the constants for the error numbers, like `ERROR_TOO_MANY_MODULES` insteads of `0xD6`

Reply:
`WARNING: LoadLibrary(D:/ID4/d3wasm/neo/build/x32-Debug/base.dll) Failed ! [193 (0xC1)]	%1 is not a valid Win32 application.`

is what it now prints with the arguement parsing. I do not really want to go there tbh. ><"